### PR TITLE
New errors images

### LINKED ISSUE #907: Usar nuevos diseños de personajes

### PR DESCRIPTION
**Pull request by fedescarpa**
Fixes #907

### CODE REVIEW

File: app/views/layouts/_submission_result_error.html.erb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+<div class="bs-callout bs-callout-broken submission-result-error" style="display: none;">
+  <h4>
+    <strong><%= t(:errored) %></strong>
+  </h4>
+  <div class="submission-result-error-body">
+    <img id="submission-result-error-animation"/>
+    <ul class="submission-result-error-body-description">
+      <li>Fijate que tu programa no tenga recursión o bucle infinito</li>
+      <li>Chequeá que tengas conexión a internet</li>
+      <li>Un texto más que hay que verlo</li>
```
Comment (fedescarpa) [inline app/views/layouts/_submission_result_error.html.erb]: @lauramangifesta, I need your review in the texts (Line 8, 9 and 10)

File: app/assets/javascripts/application/submission.js [hunk 2]
```diff
@@ -4,49 +4,77 @@ var mumuki = mumuki || {};
...
       mumuki.pin.scroll();
     }
   };
 
+  function SubmissionController() {
+    this.submitButton = $('.btn-submit');
+    this.submissionControls = $('.submission_control');
+  }
+
+  SubmissionController.prototype = {
```
Comment (flbulgarelli) [inline app/assets/javascripts/application/submission.js]: Why not `SubmitButton`?

File: app/assets/javascripts/application/submission.js [hunk 3]
```diff
@@ -4,49 +4,77 @@ var mumuki = mumuki || {};
...
       this.submissionsresultsArea.html(this.processingTemplate.html());
+      this.submissionsErrorTemplate.hide();
     },
     success: function (data) {
       this.submissionsresultsArea.html(data);
     },
-    error: function (error) {
-      this.submissionsresultsArea.html('<pre>' + error + '</pre>');
+    error: function () {
+      this.submissionsresultsArea.html('');
```
Comment (flbulgarelli) [inline app/assets/javascripts/application/submission.js]: It should be `submissionsResultsArea`

File: app/assets/javascripts/application/load-error-svg.js [hunk 4]
```diff
@@ -1,8 +1,16 @@
...
-    });
+  var svgs = ['403', '404', '500', 'timeout_1', 'timeout_2', 'timeout_3'];
+
+  mumuki.errors = mumuki.errors || {};
+
+  svgs.forEach(function (svgErrorSuffix) {
+    var image = 'error_' + svgErrorSuffix;
+    var url = '/' + image + '.svg';
+    if (!mumuki.errors[image]) {
+      $.get(url, function (data) {
```
Comment (flbulgarelli) [inline app/assets/javascripts/application/load-error-svg.js]: We need to fix this. See https://github.com/mumuki/mumuki-laboratory/issues/909

Comment (fedescarpa) [reply app/assets/javascripts/application/load-error-svg.js]: Ese arreglo va a estar en otra branch

File: app/assets/javascripts/application/submission.js (hunk 3 above)
Comment (fedescarpa) [reply app/assets/javascripts/application/submission.js]: Use el nombre que estaba, ahora lo renombro

File: app/assets/javascripts/application/submission.js (hunk 2 above)
Comment (fedescarpa) [reply app/assets/javascripts/application/submission.js]: Porque no se me ocurrió el nombre

**Comment by fedescarpa** [comment]
Yes, only the timeut_2 duration... the others are fixed. But I will do this changes when i fix the assets problems, what you think?

**Comment by flbulgarelli** [comment]
Ok

File: app/assets/javascripts/application/load-error-svg.js (hunk 4 above)
Comment (flbulgarelli) [reply app/assets/javascripts/application/load-error-svg.js]: Is this still necessary?

**Pull request merged**